frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -60,9 +71,34 @@ const LineAreaBarChart = ({
     chartSize,
   );
 
+  const keys = multipleSeries
+    .map(series => {
+      if (hasBreakoutValues(series)) {
+        return formatStaticValue(series.name, {
```

Comment:
More explanation for this line here
https://github.com/metabase/metabase/pull/26211/files#r1012746149